src/GoogleAdWordsEventForwarder.js

```diff
@@ -208,6 +282,28 @@
             return adWordEvent;
         }
 
+        function getConsentSettings() {
+            var consentSettings = {};
+
+            var googleToMpConsentSettingsMapping = {
+                ad_storage: 'adStorageConsentSDK',
+                ad_user_data: 'adUserDataConsentSDK',
+                ad_personalization: 'adPersonalizationConsentSDK',
+                analytics_storage: 'analyticsStorageConsentSDK'
```

Comment:
the `SDK` at the end of these will change based on 2 server settings being universal, and the other 2 ending in `Web` again

Reply:
triple check, but i think it's 

`adUserDataConsentSDK` --> `adUserDataConsent`
`adPersonalizationConsentSDK` --> `adPersonalizationConsent`
`adPersonalizationConsentSDK` --> `adPersonalizationConsentWeb`
`analyticsStorageConsentSDK` --> `analyticsStorageConsentWeb`